tests/cassettes/file_for_download.json

```diff
@@ -0,0 +1 @@
+{"recorded_with": "betamax/0.8.0", "http_interactions": []}
```

Comment:
This doesn't seem to actually have the data for the test.

Reply:
Believe me or not I wasn't seen that 👍 Thanks.

Reply:
No worries :)